README.md

```diff
@@ -30,6 +30,31 @@ a test which verifies your change.
   $ mvn clean install
 ```
 
+## Jenkin Setup
+
+### Using Multibranch Pipeline`
+
+Create a new `Multibranch Pipeline` item.
+
+Select `Branch Source` of type `Gerrit`.
+
+Specify project repository URL, only `http` or `https` based protocol is
+supported at this point, copy the URL from project settings at Gerrit.
+
+Trigger `Scan Multibranch Pipeline now` either manually or by external
+trigger.
+
+Notice the `Changes` tab at the job information, per each review an entry will
+be created.
+
+### Using Environment Variables
+
+|Key                  |Description                                                     |
+|---------------------|----------------------------------------------------------------|
+|GERRIT_API_URL       |Gerrit API URL, onlhy `http` and `https` protocols are supported|
+|GERRIT_CREDENTIALS_ID|Jenkins credentials object id                                   |
+|BRANCH_NAME          |Gerrit reference name nn/nnnn/n                                 |
```

Comment:
And GERRIT_PROJECT as well with the name of project that the change is referring to.